src/__experimental__/components/radio-button/radio-button.stories.js

```diff
@@ -20,6 +20,7 @@ function makeStory(name, themeSelector) {
       >
         <RadioButton
           id='input-1'
+          checked
```

Comment:
maybe lets use checked="checked" ?

Reply:
The PropType is a bool though and the html5 spec supports just using checked at level